src/api/CLAUDE.md

```
@@ -0,0 +1 @@
+This project uses AGENTS.md files for AI context. Read AGENTS.md in this directory for relevant instructions and knowledge.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)